### DISCUSSION TITLE
Virtual camera using recordings for debug

### DISCUSSION THREAD

**Discussion by ripvega** [blakeblackshear/frigate#3653] [Ideas]
I'll be really surprised if this hasn't been suggested before but I searched and couldn't find it.

Could Frigate have an option to use a recording to set zones and fine-tune the config rather than rely just on a live camera?

I have a long driveway to monitor and only get a few vehicles a day.

Using a recording of vehicles would make the process much faster and I could finally tune in the detection of the once a day red mail delivery van!

**Comment by NickM-27** [comment]
It can be done using https://docs.frigate.video/contributing#2-create-a-local-config-file-for-testing but may be built it an some point in the future.

**Comment by ripvega** [comment]
Brilliant! Thank you.

But can I do this running Frigate as HA addon?

**Comment by ripvega** [comment]
I got it working. A really great utility. Thank you.
